grammar.go

```
@@ -0,0 +1,522 @@
+	return ""
+}
+
+func (r *Grammar) CompileIndex(blueprint contractsschema.Blueprint, command *contractsschema.Command) string {
+	return fmt.Sprintf("create index %s on %s (%s)",
+		r.wrap.Column(command.Index),
+		r.wrap.Table(blueprint.GetTableName()),
+		r.wrap.Columnize(command.Columns),
+	)
+}
+
+func (r *Grammar) CompileIndexes(schema, table string) (string, error) {
```

golangci-lint (1.62.2)

SA4009: argument schema is overwritten before first use

(staticcheck)